test/PublicApiTests/BaseLine/Microsoft.OData.PublicApi.netstandard2.0.bsl

```diff
@@ -7925,7 +7925,6 @@ public enum Microsoft.OData.Client.SaveChangesOptions : int {
     BatchWithSingleChangeset = 1
     BulkUpdate = 128
     ContinueOnError = 2
-    DeepInsert = 256

```

Comment:
why this change?

Reply:
One of previous commit forgot to update the bsl.